api/utils/settings.py

```diff
@@ -29,8 +29,14 @@ class Settings(BaseSettings):
     MAIL_PORT: int = config('MAIL_PORT')
     MAIL_SERVER: str = config('MAIL_SERVER')
 
+    # FACEBOOK_APP_ID: str = config("FACEBOOK_APP_ID")
+    # FACEBOOK_APP_SECRET: str = config("FACEBOOK_APP_SECRET")
+
+    # FLUTTERWAVE_SECRET: str = config("FLUTTERWAVE_SECRET")
+
```

Comment:
you commented this out

Reply:
I commented out only Flutterwave secret pending when I get the  Devops team to add it to the production environment. It is commented out to make sure it does not break anything. @Goketech

Reply:
@Goketech The devops team has added the flutterwave secrets to the server.